sos/report/plugins/kernel.py

```diff
@@ -107,6 +107,10 @@ def setup(self):
             "/var/log/dmesg",
             "/sys/fs/pstore",
             "/sys/kernel/debug/dynamic_debug/control",
+            "/proc/pagetypeinfo",
```

Comment:
We collect this in `memory` plugin; we can not collect it in two plugins as that cause some issues during collection. If you think the file is worth to collect here, please provide a rationale and remove the file from `memory` plugin.

Reply:
ah yes..sorry forgot to check this..will update and send new version of patch